Rocket.Chat/Views/Chat/New Chat/ChatItems/ImageMessageChatItem.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ImageMessageChatItem.swift
+//  Rocket.Chat
+//
+//  Created by Rafael Streit on 01/10/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import Foundation
+import DifferenceKit
+import RocketChatViewController
+
+struct ImageMessageChatItem: ChatItem, Differentiable {
+    var relatedReuseIdentifier: String {
+        return ImageMessageCell.identifier
+    }
+
+    var identifier: String
+    var title: String?
+    var descriptionText: String?
+    var imageURL: URL?
+
+    var differenceIdentifier: String {
+        return identifier
+    }
+
+    func isContentEqual(to source: ImageMessageChatItem) -> Bool {
+        return
+            identifier == source.identifier &&
```

Comment:
I confirmed and we shouldn't check for identity here, just for content equality.